File: packages/cli/test/watcher/lambda.go
package main

import (
  "github.com/aws/aws-lambda-go/events"
  "github.com/aws/aws-lambda-go/lambda"
)


func Handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
  return events.APIGatewayProxyResponse{Body: "Hi! I'm root!", StatusCode: 200}, nil
}

func main() {
  lambda.Start(Handler)
}

File: packages/cli/test/playground-region/src/lambda.js
export async function main() {
  const body = ["hi"];
  //  for (let i = 0; i < 100000; i++) {
  //    body.push("1234567890");
  //  }

  return {
    statusCode: 200,
    body: body.join(""),
  };
}
